Android.bp

```diff
@@ -0,0 +1,143 @@
+        "Source/GmmLib/CachePolicy/GmmGen9CachePolicy.cpp",
+        "Source/GmmLib/CachePolicy/GmmGen10CachePolicy.cpp",
+        "Source/GmmLib/CachePolicy/GmmGen11CachePolicy.cpp",
+        "Source/GmmLib/CachePolicy/GmmGen12CachePolicy.cpp",
+        "Source/GmmLib/CachePolicy/GmmXe_LPGCachePolicy.cpp",
+        "Source/GmmLib/CachePolicy/GmmXe2_LPGCachePolicy.cpp",
+        "Source/GmmLib/CachePolicy/GmmGen12dGPUCachePolicy.cpp",
+        "Source/GmmLib/Platform/GmmGen11Platform.cpp",
+        "Source/GmmLib/Platform/GmmGen12Platform.cpp",
+        "Source/GmmLib/Platform/GmmGen8Platform.cpp",
+        "Source/GmmLib/Platform/GmmGen9Platform.cpp",
+        "Source/GmmLib/Platform/GmmGen10Platform.cpp",
+        "Source/GmmLib/Platform/GmmPlatform.cpp",
+        "Source/GmmLib/Resource/GmmResourceInfo.cpp",
+        "Source/GmmLib/Resource/GmmResourceInfoCommon.cpp",
+        "Source/GmmLib/Resource/GmmResourceInfoCommonEx.cpp",
+        "Source/GmmLib/Resource/GmmRestrictions.cpp",
+        "Source/GmmLib/Resource/Linux/GmmResourceInfoLinCWrapper.cpp",
+        "Source/GmmLib/Texture/GmmGen7Texture.cpp",
+        "Source/GmmLib/Texture/GmmGen8Texture.cpp",
+        "Source/GmmLib/Texture/GmmGen9Texture.cpp",
+        "Source/GmmLib/Texture/GmmGen10Texture.cpp",
+        "Source/GmmLib/Texture/GmmGen11Texture.cpp",
+        "Source/GmmLib/Texture/GmmGen12Texture.cpp",
+        "Source/GmmLib/Texture/GmmXe_LPGTexture.cpp",
+        "Source/GmmLib/Texture/GmmTexture.cpp",
+        "Source/GmmLib/Texture/GmmTextureAlloc.cpp",
+        "Source/GmmLib/Texture/GmmTextureSpecialCases.cpp",
+        "Source/GmmLib/Texture/GmmTextureOffset.cpp",
+        "Source/GmmLib/GlobalInfo/GmmInfo.cpp",
+        "Source/GmmLib/Utility/CpuSwizzleBlt/CpuSwizzleBlt.c",
+        "Source/GmmLib/Utility/GmmLog/GmmLog.cpp",
+        "Source/GmmLib/Utility/GmmUtility.cpp",
+        "Source/GmmLib/TranslationTable/GmmAuxTable.cpp",
+        "Source/GmmLib/TranslationTable/GmmPageTableMgr.cpp",
+        "Source/GmmLib/TranslationTable/GmmUmdTranslationTable.cpp",
+        "Source/GmmLib/GlobalInfo/GmmClientContext.cpp",
+        "Source/GmmLib/GlobalInfo/GmmLibDllMain.cpp",
+    ],
+
+    cflags: [
+        "-Werror",
+        "-Wno-logical-op-parentheses",
+        "-Wno-shift-negative-value",
+        "-Wno-unused-parameter",
+        "-DGMM_LIB_DLL",
+        "-DGMM_LIB_DLL_EXPORTS",
+        "-DGMM_UNIFIED_LIB",
+        "-DGMM_UNIFY_DAF_API",
+        "-DISTDLIB_UMD",
+        "-DSMALL_POOL_ALLOC",
+        "-DUNUSED_ISTDLIB_MT",
+        "-D_ATL_NO_WIN_SUPPORT",
+        "-D_X64",
+        "-D__GFX_MACRO_C__",
+        "-D__GMM",
+        "-D__STDC_CONSTANT_MACROS",
+        "-D__STDC_LIMIT_MACROS",
+        "-D__UMD",
+        "-march=corei7",
+        "-fvisibility=hidden",
+        "-fno-omit-frame-pointer",
+        "-pthread",
+    ],
+
+    cppflags: [
+        "-Wno-implicit-fallthrough",
+        "-Wno-missing-braces",
+        "-Wno-parentheses-equality",
```

Comment:
I tried building without the following flags that I didn't find in the current system, and it works. Consider removing them from here:

- `-Wno-parentheses-equality`
- `-Wno-logical-not-parentheses`
- `-Wno-missing-field-initializers`